articles/azure-maps/migrate-from-bing-maps-web-services.md

```diff
@@ -528,39 +526,39 @@ The following table cross-references the Bing Maps API parameters with the compa
 
 Point of interest data can be searched in Bing Maps by using the following APIs:
 
-* **Local search:** Searches for points of interest that are nearby (radial search), by name, or by entity type (category). The Azure Maps [POI search](/rest/api/maps/search/getsearchpoi) and [POI category search](/rest/api/maps/search/getsearchpoicategory) APIs are most like this API.
-* **Location recognition**: Searches for points of interests that are within a certain distance of a location. The Azure Maps [nearby search](/rest/api/maps/search/getsearchnearby) API is most like this API.
-* **Local insights:** Searches for points of interests that are within a specified maximum driving time or distance from a specific coordinate. This is achievable with Azure Maps by first calculating an isochrone and then passing it into the [search within geometry](/rest/api/maps/search/postsearchinsidegeometry) API.
+* **Local search:** Searches for points of interest that are nearby (radial search), by name, or by entity type (category). The Azure Maps [POI search] and [POI category search] APIs are most like this API.
```

Comment:
In line 529, the "POI category search" link doesn't render properly. Same for all other instances of the phrase in this article.